docsource/modules150-160.rst

```diff
@@ -786,7 +786,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| transifex                                 |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| uom                                             |                      |                                                 |
+| uom                                             | Nothing to do        |No DB layout changes                                                 |
```

Comment:
Don't write `No DB layout changes` here. This is put by the module. And if you put it, remove the extra spaces for proper RST syntax.

Reply:
Well noted for the `No DB layout changes`. And thank you for spotting the spaces issue. :)

I removed the `No DB layout changes`, and fixed the spaces.